inc/html.class.php

```diff
@@ -5372,5 +5364,85 @@ static function showCheckboxMatrix(array $columns, array $rows, array $options=a
    }
 
 
+
+   /**
+    * List of js lib to load
+    */
+   static function core_js_lib() {
+       return array(
+           "lib/jquery/js/jquery-1.10.2.js",
+           "lib/jquery/js/jquery-ui-1.10.4.custom.js",
+//           "lib/tiny_mce/tiny_mce_src.js",
+           // Plugins jquery
+           "lib/jqueryplugins/backtotop/BackToTop.jquery.js",
+           "lib/jqueryplugins/select2/select2.js",
+           "lib/jqueryplugins/qtip2/jquery.qtip.js",
+           "lib/jqueryplugins/jstree/jquery.jstree.js",
+           "lib/jqueryplugins/rateit/jquery.rateit.min.js",
+           "lib/jqueryplugins/jquery-ui-timepicker-addon/jquery-ui-timepicker-addon.js",
+           "lib/jqueryplugins/jquery-file-upload/js/jquery.iframe-transport.js",
+           "lib/jqueryplugins/jquery-file-upload/js/jquery.fileupload.js",
+           "lib/jqueryplugins/jcrop/jquery.Jcrop.js",
+           "lib/jqueryplugins/imagepaste/jquery.image_paste.js",
+           "lib/jqueryplugins/spectrum-colorpicker/spectrum.js",
+           "lib/jqueryplugins/jquery-gantt/js/jquery.fn.gantt.js",
+           "lib/jquery/i18n/jquery.ui.datepicker-[LANG].js",
+           "lib/jqueryplugins/jquery-ui-timepicker-addon/i18n/jquery-ui-timepicker-[LANG].js",
+           "lib/jqueryplugins/select2/select2_locale_[LANG].js",
+           // Some Javascript-Functions which we may need later
+           "script.js"
+       );
+   }
+
+
+
+   /**
+    * Display an image from sprite
+    */
+   static function sprite_img($name, $options=array()) {
+
+      if (!isset($options['title'])) {
+         $options['title'] = '';
+      }
+
+      $url = false;
+      if (!empty($options['url'])) {
+         $url = $options['url'];
+         unset($options['url']);
+      }
+
+      $prefix = 'glpimg';
+      if (isset($options['prefix'])) {
+          $prefix = $options['prefix'];
+      }
+
+      $addclass = '';
+      if (isset($options['addclass'])) {
+          $addclass = $options['addclass'];
+      }
+
+      $id = '';
+      if (isset($options['id'])) {
+          $id = "id='".$options['id']."'";
+      }
+
+      $onclick = '';
+      if (isset($options['onclick'])) {
+          $onclick = "onclick='".$options['onclick']."'";
+      }
+
+      $style = '';
+      if (isset($options['style'])) {
+          $onclick = "style='".$options['style']."'";
+      }
+
+      $image = sprintf("<span class='block %s %s-%s %s' %s %s %s %s></span>",
```

Comment:
I was expecting here a 

if (file_exists(main.png))
     use the sprite
else
     use the image

and main.png + man.css being generated during the release process.